src/ast/query.rs

```diff
@@ -556,6 +563,8 @@ pub struct Cte {
     pub query: Box<Query>,
     pub from: Option<Ident>,
     pub materialized: Option<CteAsMaterialized>,
+    // needed for accurate span reporting
```

Comment:
can we update the comment to rather say what's being pointed to (thinking all attached tokens are implicitly needed for span reporting so that no need to treat this case specially)?

Reply:
yup 👍